backend/app/api/diagnosis.py

```
@@ -0,0 +1,252 @@
+            "id": m.id,
+            "role": m.role,
+            "content": m.content,
+            "created_at": m.created_at.isoformat()
+        } for m in messages
+    ]
+
+@router.post("/history/{history_id}/chat")
+async def add_chat_message(
+    history_id: int,
+    payload: dict,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)